solc/CommandLineInterface.cpp

```diff
@@ -770,7 +770,10 @@ bool CommandLineInterface::processInput()
 	}
 
 	m_compiler.reset(new CompilerStack(fileReader));
+
 	auto scannerFromSourceName = [&](string const& _sourceName) -> solidity::Scanner const& { return m_compiler->scanner(_sourceName); };
+	auto formatter = SourceReferenceFormatter(cerr, scannerFromSourceName);
```

Comment:
I would prefer the (actually shorter) and more explicit version:

`SourceReferenceFormatter formatter(cerr, scannerFromSourceName);`